packages/fxa-content-server/app/tests/spec/views/subscriptions_product_redirect.js

```diff
@@ -72,16 +71,33 @@ describe('views/subscriptions_product_redirect', function() {
 
   describe('render', () => {
     it('renders, initializes flow metrics, navigates to payments server', () => {
-      assert.lengthOf(view.$('.subscriptions-redirect'), 1);
-      assert.isTrue(view.initializeFlowEvents.calledOnce);
-      assert.deepEqual(PaymentServer.navigateToPaymentServer.args, [
-        [
-          view,
-          config.subscriptions,
-          `products/${PRODUCT_ID}`,
-          { plan: 'plk_12345' },
-        ],
-      ]);
+      return render().then(() => {
```

Comment:
This seems unrelated to changes signup code changes?

Reply:
It's needed to pass this change here: https://github.com/mozilla/fxa/pull/3477/files#diff-77c3a2279a222015eab66c6e7c1e8c9dL29